### ISSUE TITLE
Can you give us an example image folder?

### ISSUE THREAD

**Issue by XiangruiZeng** [theduynguyen/Keras-FCN#1]
Hi, I am new to this. I will appreciate it if you can give me an example image folder that I can run directly.

**Comment by theduynguyen** [comment]
Hi,

first of all: sorry for the late answer and the lack of documentation. I have been a bit busy lately. 

Please check out the latest commit. I have added a script called "COCO2Seg.py". You have to adjust this script according to your dataset paths. In particular, you have to change the following variables:

- coco_dir: the path where your train2014, val2014 and annotations directories lies
- out_dir: the path where you want to have your input data for the models
- train: if True, you will create training data - if False, you will create validation data
- n_samples: the number of images in the folders the script is going to take as input
- seg_labels: which labels are considered for the segmentation task

Afterwards, you can start the script and take the out_dir path as your input for the training.
Hopefully this helps. If you have questions feel free to ask.
